config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-presubmit-1.27.yaml

```
@@ -42,16 +42,15 @@ presubmits:
     labels:
       preset-dind-enabled: "true"
       preset-kind-volume-mounts: "true"
-      preset-azure-cred-only: "true"
-      preset-azure-anonymous-pull: "true"
-      preset-azure-capz-sa-cred: "true"
+      preset-azure-community: "true"
     extra_refs:
     - org: kubernetes-sigs
       repo: cluster-api-provider-azure
-      base_ref: release-1.15
+      base_ref: release-1.16
       path_alias: sigs.k8s.io/cluster-api-provider-azure
       workdir: true
     spec:
+      serviceAccountName: azure
```

I think these are also missing this?
```suggestion
      cluster: eks-prow-build-cluster
      serviceAccountName: azure
```

The `cluster` doesn't come for free with the `preset-azure-community` label.

---

According to the master jobs, cluster should be in the same level of job name, not in spec?

---

Ah yes, you're right.